pulpcore/cli/container/context.py

```diff
@@ -101,10 +103,13 @@ class PulpContainerRepositoryContext(PulpRepositoryContext):
     UPDATE_ID = "repositories_container_container_partial_update"
     DELETE_ID = "repositories_container_container_delete"
     SYNC_ID = "repositories_container_container_sync"
+    TAG_ID: ClassVar[str] = "repositories_container_container_tag"
+    UNTAG_ID: ClassVar[str] = "repositories_container_container_untag"
```

Comment:
This looks like a rebase is needed.

Reply:
Are you talking about your new PR refactoring the IDs? #448 I can wait till after it's merged to rebase.

Reply:
Oh, yes, i get confused what's merged or not.